testsuite/run_sets/secondary_parallelizable.yml

```diff
@@ -65,6 +65,7 @@
 - features/secondary/min_salt_pkgset_beacon.feature
 - features/secondary/min_custom_pkg_download_endpoint.feature
 - features/secondary/min_monitoring.feature
+- features/secondary/min_ansible_control_node.feature
```

Comment:
If we will merge this, but the feature is not completed, I think it better if we comment that line for now.

Reply:
Well, no. The merge will go just into another branch where we run this in the testnaica environment.
This is the place where all parts of the feature will come together. Only when this is usable, it will be merged to master.

Reply:
As already clarified, we're good enabling this now since this PR is not targetting `master` branch.